### PR TITLE
Feature/persist fta diagram layout

### PR DESCRIPTION
**Pull request by kostobog**
Partially implements #131.
Server side related changes:
- kbss-cvut/fta-fmea#54

### CODE REVIEW

**Review by blcham — CHANGES_REQUESTED** [review]
see my comments about @ts-ignore

**Comment by blcham** [comment]
@kostobog

File: src/components/editor/faultTree/shapes/RenderTree.tsx [hunk 1]
```diff
@@ -26,8 +27,13 @@ const renderTree = (container, node, parentShape = null) => {
         nodeShape.attr(['probabilityLabel', 'text'], node.probability.toExponential(2));
     }
     // @ts-ignore
-    nodeShape.set('custom/faultEventIri', node.iri)
-
+    nodeShape.set(JOINTJS_NODE_MODEL.faultEventIri, node.iri)
+    const r = node.rectangle;
+    if(r &&  r.x && r.y && r.width && r.height){
+        nodeShape.position(node.rectangle.x, node.rectangle.y);
+        // @ts-ignore
```
[the remark this answers is not in the document]
Comment (kostobog) [reply src/components/editor/faultTree/shapes/RenderTree.tsx:34]: Yes, without it I get a compilation error.

File: src/components/editor/faultTree/Editor.tsx [hunk 2]
```diff
@@ -79,6 +81,22 @@ const Editor = ({setAppBarName}: DashboardTitleProps) => {
         hideHighlightedBorders();
     }
 
+    const handleMoveEvent = (elementView, evt) => {
+        const faultEventIri = elementView.model.get(JOINTJS_NODE_MODEL.faultEventIri);
+        // @ts-ignore
```
[the remark this answers is not in the document]
Comment (kostobog) [reply src/components/editor/faultTree/Editor.tsx:86]: In Editor, the @ts-ignore is used when working with `_localContext`. I resolved this by encapsulating access to `_localContext` in getters and refactored the rest of the annotated lines.

I also removed other @ts-ignore which did not cause a compilation error.

**Pull request merged**